src/app/Models/Comment.php

```diff
@@ -18,11 +20,27 @@
  * @property DateTime updated_at
  * @property string user_id
  * @property int revision_id
+ * @property int review_num
  */
 class Comment extends Model
 {
     use HasFactory, ApiResource;
 
+    private const review = [ 'DECLINED', 'REQUEST CHANGES', 'APPROVED' ];
```

Comment:
Constants are always in uppercase. Also, since this is an array, this should be in plural: `REVIEWS`